sympy/printing/latex.py

```diff
@@ -209,6 +209,10 @@ def __init__(self, settings=None):
     def _add_parens(self, s):
         return r"\left({}\right)".format(s)
 
+    # TODO: merge this with the above, which requires a lot of test changes
+    def _add_parens_lspace(self, s):
+        return r"\left( {}\right)".format(s)
```

Comment:
This is a bit ugly, but I really don't want to sift through the 5 occurences of `\left(` in the tests for whether they go through the tuple printer, as it would add tonnes of noise to this PR.

Is the presence of this space deliberate?

Reply:
My understanding of LaTeX is that the space should never be needed, but I could be wrong. You might want to find a reference one way or another (perhaps ask on tex.stackexchange). If the space isn't needed there's no reason the put it there. 

IMO we should aim to make the the output of `latex()` both human and machine readable, but we don't really do that right now. But even then, I'm not really sure if the space here helps with that.